packages/driver/src/cy/net-stubbing/events/before-request.ts

```diff
@@ -302,7 +302,10 @@ export const onBeforeRequest: HandlerFn<CyHttpMessages.IncomingRequest> = (Cypre
     resolve = _resolve
   })
 
-  request.setLogFlag = Cypress.ProxyLogging.logInterception(request, route).setFlag
+  request.setLogFlag = Cypress.RequestLogs['getFlagSetter'](request, route)
+
+  // consider a function to be 'spying' until it actually stubs/modifies the response
+  request.setLogFlag?.(!_.isNil(route.handler) && !_.isFunction(route.handler) ? 'stubbed' : 'spied')
```

Comment:
```suggestion
  const flag = !_.isNil(route.handler) && !_.isFunction(route.handler) ? 'stubbed' : 'spied'
  request.setLogFlag?.(flag)
```

Took me quite a while to parse this line.

Reply:
Cleaned this up in a79b9435f01bcfa69f20348b001a449164ae91ae